blackbox/docs/enterprise/monitoring.txt

```diff
@@ -0,0 +1,106 @@
+.. _jmx_monitoring_plugin:
+
+=====================
+JMX Monitoring Plugin
+=====================
+
+The CrateDB monitoring plugin exposes query metrics via the `JMX`_ API.
+
+This plugin is a component of the `Enterprise Edition`_ of CrateDB.
+
+To use this plugin, you need to:
+
+- Set ``license.enterprise`` in the CrateDB settings
+- Enable stats
+- Enable the JMX API
```

Comment:
would make a ref to https://github.com/crate/crate/pull/4950/files#diff-3634480cb27984b6a13e0e9805b4e6caR26